java-jacoco/src/main/java/org/sonar/plugins/jacoco/JacocoReportReader.java

```diff
@@ -118,9 +121,20 @@ public CoverageBuilder analyzeFiles(ExecutionDataStore executionDataStore, Colle
         analyzeClassFile(analyzer, classFile);
       }
     }
+    logNoMatchClasses(coverageBuilder.getNoMatchClasses());
     return coverageBuilder;
   }
 
+  private void logNoMatchClasses(Collection<IClassCoverage> noMatchClasses) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "logNoMatchClasses" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1953&issues=AWJTUsi8orNz2_dLi7nx&open=AWJTUsi8orNz2_dLi7nx)

[ISSUE_KEY]: AWJTUsi8orNz2_dLi7nx